src/loaders/libcublas.c

```diff
@@ -41,21 +45,22 @@ int load_libcublas(int major, int minor, error *e) {
 
 #if defined(WIN32) || defined(_WIN32) || defined(WIN64) || defined(_WIN64)
   {
-    static const char DIGITS[] = "0123456789";
-    char libname[] = "cublas64_??.dll";
+    const char* libname_pattern = "cublas64_%d%d.dll";
+    char libname[64];
 
-    libname[9] = DIGITS[major];
-    libname[10] = DIGITS[minor];
+    #ifdef DEBUG
+    fprintf(stderr, "Loading cuBLAS %d.%d.\n", major, minor);
+    #endif
+    sprintf(libname, libname_pattern, major, minor);
```

Comment:
I object to sprintf.  Use snprintf at the minimum.

Also check for errors and report them.  We don't want to use a truncated name for loading.

Finally, have the format string in the arguments to snprintf().